### DISCUSSION TITLE
How can i make a page explicitly dark mode always?

### DISCUSSION THREAD

**Discussion by RanitManik** [heroui-inc/heroui#4144] [Q&A] [answered]
i am using dark and light mode from NextThemesProvider and want to make a page explicitly dark mode always

**Comment by pavitra-infocusp** [comment]
Set [`defaultTheme`](https://nextui.org/docs/customization/theme#api-reference:~:text=false-,defaultTheme,-light) to dark on the NextUI plugin in the `tailwind.config.js` file.

**Comment by RanitManik** [comment]
i wanted to know if there is a way that i can make a page always dark or always light irrelevent of the the defaultTheme and current Theme

**Reply by pavitra-infocusp** [reply, marked as answer]
you can set "dark" class to html or body tag.
